plugins/node/opentelemetry-instrumentation-ioredis/README.md

```diff
@@ -22,25 +22,31 @@ npm install --save @opentelemetry/instrumentation-ioredis
 
 ## Usage
 
-To load a specific instrumentation (**ioredis** in this case), specify it in the Node Tracer's configuration
+To load a specific instrumentation (**ioredis** in this case), specify it in the registerInstrumentations's configuration
 
 ```javascript
 const { NodeTracerProvider } = require('@opentelemetry/node');
 const { IORedisInstrumentation } = require('@opentelemetry/instrumentation-ioredis');
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
 
-const provider = new NodeTracerProvider({
-  // be sure to disable old plugin
-  plugins: {
-    ioredis: { enabled: false, path: '@opentelemetry/plugin-ioredis' }
-  },
-});
+const provider = new NodeTracerProvider();
 provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
 provider.register();
 
-const ioredisInstrumentation = new IORedisInstrumentation({
-  // see under for available configuration
-});
-
+registerInstrumentations({
+  instrumentations: [
+    new IORedisInstrumentation({
+      // see under for available configuration
+    }),
+    {
+      // be sure to disable old plugin but only if it was installed
+      plugins: {
+        ioredis: { enabled: false, path: '@opentelemetry/plugin-ioredis' }
+      },
+    }
+  ],
+  tracerProvider: provider,
```

Comment:
I would remove this option here as it is usually correct and best to let instrumentations pick up the global `TracerProvider`. I know there are special usecases where this is not intended but most users want the global and I think Readme should show the usual case and docs of instrumentation packages all the details/special cases.

This comment is for sure not blocking, so feel free to keep it as it is.

Reply:
I was really struggling with this if either keep it or not, then I aligned this across all so user has better understanding that it can be done like this. We will have to make one more round with documentation anyway when we remove plugins, we can decide then again, but it was already many changes here. I have chosen to add it after all so that it is easy to understand that this is possible. But we might add maybe some more examples or explanation too that it can be done differently. For now this code will work for 100%